lib/helpers.ts

```diff
@@ -128,3 +77,20 @@ export function getAuthorizationClaimItems(
     : authInfo.token[claim];
   return Array.isArray(value) ? value : [];
 }
+
+/**
+ * Add hooks to an existing core-sdk config
+ */
+export const addHooks = <Config extends CoreSdkConfig>(
```

Comment:
this code was duplicated from web-js-sdk

might worth putting it in core-sdk with the rest of the wrapwith there

Reply:
https://github.com/descope/core-js-sdk/pull/147

Reply:
done